features/htcondor/client/service.pan

```diff
@@ -1,29 +1,33 @@
 unique template features/htcondor/client/service;
 
 
-variable CONDOR_CONFIG={
-  #Define the appropriate config files
-  if(!is_defined(SELF['cfgfiles'])){
-    SELF['cfgfiles']=list();
-  };
+variable CONDOR_CONFIG = {
+    # Define the appropriate config files
+    if (!is_defined(SELF['cfgfiles'])) {
+        SELF['cfgfiles'] = list();
+    };
 
-  # global & security mandatory
-  file_list=list('global','security');
-  foreach(i;file;file_list){
-    num = length( SELF['cfgfiles']);
-    SELF['cfgfiles'][num] = nlist( 'name',file,'contents','features/htcondor/templ/' + file);
-  };
+    # global & security mandatory
+    file_list = list('global', 'security');
+    foreach (i; file; file_list) {
+        num = length( SELF['cfgfiles']);
+        SELF['cfgfiles'][num] = dict(
```

Comment:
this can be replaced by `append`

Reply:
I would suggest that the maintainer of the htcondor templates are performing further modifications on this feature, as I cannot test it in production.